polaris/ocean/tasks/cosine_bell/analysis.py

```diff
@@ -44,135 +43,64 @@ def __init__(self, component, resolutions, icosahedral, subdir,
         dependencies : dict of dict of polaris.Steps
             The dependencies of this step
         """
-        super().__init__(component=component, name='analysis', subdir=subdir)
-        self.resolutions = resolutions
-        self.icosahedral = icosahedral
-
-        for resolution in resolutions:
-            mesh_name = resolution_to_subdir(resolution)
-            base_mesh = dependencies['mesh'][resolution]
-            init = dependencies['init'][resolution]
-            forward = dependencies['forward'][resolution]
-            self.add_input_file(
-                filename=f'{mesh_name}_mesh.nc',
-                work_dir_target=f'{base_mesh.path}/base_mesh.nc')
-            self.add_input_file(
-                filename=f'{mesh_name}_init.nc',
-                work_dir_target=f'{init.path}/initial_state.nc')
-            self.add_input_file(
-                filename=f'{mesh_name}_output.nc',
-                work_dir_target=f'{forward.path}/output.nc')
-
-        self.add_output_file('convergence.png')
-
-    def run(self):
-        """
-        Run this step of the test case
+        convergence_vars = {0: {'name': 'tracer1',
+                                'title': 'tracer1',
+                                'units': '',
+                                'zidx': 0}}
```

Comment:
Based on this example, it seems like `convergence_vars` should be a `list` of `dict`, rather than a nested `dict`.  The key `0` is never used for anything.